users/api/http/transport.go

```diff
@@ -53,13 +56,34 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 		opts...,
 	))
 
+	mux.Post("/password/request", kithttp.NewServer(
+		kitot.TraceServer(tracer, "res-req")(passwordResetRequestEndpoint(svc)),
+		decodePasswordResetRequest,
+		encodeResponse,
+		opts...,
+	))
+
+	mux.Put("/password/reset", kithttp.NewServer(
+		kitot.TraceServer(tracer, "reset")(passwordResetPutEndpoint(svc)),
+		decodePasswordReset,
+		encodeResponse,
+		opts...,
+	))
+
 	mux.Post("/tokens", kithttp.NewServer(
 		kitot.TraceServer(tracer, "login")(loginEndpoint(svc)),
 		decodeCredentials,
 		encodeResponse,
 		opts...,
 	))
 
+	mux.Get("/users", kithttp.NewServer(
+		kitot.TraceServer(tracer, "register")(userInfoEndpoint(svc)),
```

Comment:
I think that you should change `register` string to `user_info` or something like that.